### PR TITLE
feat: add phone and email filters to planters

### LINKED ISSUE #111: Add planter identifier to Planters filter
We need the ability to filter planters by planter identifier (`planter_identifier` in the database and `planterIdentifier` in the API response object). This will either be an email address or a phone number (depending on the planter).

Screenshot 2021-06-09 at 20 01 52

### PR DESCRIPTION
**Pull request by VLuisa**
Resolves #111 

image
image

### CODE REVIEW

File: src/models/FilterPlanter.js [hunk 1]
```diff
@@ -33,6 +33,16 @@ export default class Filter {
...
     }
 
+    if (this.email) {
+      where.email = {
+        ilike: this.email,
+      };
+    }
+
+    if (this.phone) {
+      where.phone = this.phone;
```
Comment (nmcharlton) [inline src/models/FilterPlanter.js:43]: I think this should use `ilike` as well for consistency with the other string filters.

**Review by nmcharlton — CHANGES_REQUESTED** [review]
Just one suggestion.

File: src/models/FilterPlanter.js (hunk 1 above)
Comment (VLuisa) [reply src/models/FilterPlanter.js:43]: Great point

**Review by nmcharlton — APPROVED** [review]
Nice work 👍

**Pull request merged**